README.md

```diff
@@ -23,32 +23,7 @@ installed. This helps with making the builds reproducible and the provenances
 verifiable. The toml file should conform to the `BuildConfig` structure defined
 in the [`common`](/common/) package.
 
-The [`cmd/build`](/cmd/build/) command line tool described above can be used
-for building and releasing the binaries, and at the same time for generating a
-corresponding provenance file. To use this tool, the developers need to provide
-a toml file similar to the one in [`testdata/build.toml`](/testdata/build.toml).
-See the definition of `BuildConfig` in package [`common`](/common/) for the
-description of each field.
-
-The following command can then be used to build the binary and generate the
-provenance file:
-
-```bash
-$ bazel run  //cmd/build:main -- \
-  -config <path-to-transparent-release>/testdata/build.toml \
-```
-
-This involves fetching the sources from the Git repository specified in the
-config file. It is also possible to perform the release from an already existing
-local repository, by specifying `-git_root_dir`. In this case, the binary will
-be built from the repo, only if the latest commit matches the one specified in
-the config file.
-
-```bash
-$ bazel run  //cmd/build:main -- \
-  -config <path-to-transparent-release>/testdata/build.toml \
-  -git_root_dir <path-to-git-repo-root>
-```
+Check the [`development guidelines`](docs/development-guidelines.md) for a quick start to [`building and provenances`](docs/development-guidelines.md#building-and-provenances).
```

Comment:
These commands are meant to be used by end-users not just the developers. So I am a bit unsure that it is a good idea to refer the users to the development guidelines for how to use the commands.

Reply:
Moved back to README and added a link to the development guidelines for building.